### ISSUE TITLE
"SCRIPT445: Object doesn't support this action" error with TinyMCE on IE/Edge

### ISSUE THREAD

**Issue by netdelight** [thedevdojo/voyager#2858]
- Laravel Version: 5.4.36
- Voyager Version: 1.0.7
- PHP Version: 7.1
- Database Driver & Version:MySQL 5.1

### Description:
On IE (Edge and 11), TinyMCE is not loading and the error "SCRIPT445: Object doesn't support this action" appears in the console.

### Steps To Reproduce:
Display Voyager admin and edit a page in Edge.

A workaround is to add this before the app.js is loaded :
```
<script type="text/javascript">
        window.tinymce = { suffix: "", base: "{{ voyager_asset('js') }}" }
</script>
```
(based on this solution : 
https://stackoverflow.com/questions/23732815/tinymce-does-not-discover-correct-plug-in-path-in-ie10-and-chrome )

**Comment by emptynick** [comment]
Judging by [this issue](https://github.com/tinymce/tinymce/issues/1096) it was resolved in a newer version and will then be fixed by https://github.com/the-control-group/voyager/pull/3832

**Comment by emptynick** [comment]
Closing as #3832 was merged (into 1.x, to be released in a future 1.2 release)
